waku/v2/protocol/waku_swap/waku_swap_types.nim

```diff
@@ -21,6 +21,8 @@ type
     mode* : SwapMode
     paymentThreshold* : int
     disconnectThreshold* : int
+    stateUpdateOverRide*: bool
```

Comment:
What is stateUpdateOveride? Not obvious to me. Perhaps add a comment?

Reply:
Added a comment. It indicates whether or not the account state of the peer will proceed to be updated be updated if there was an issue redeeming a cheque.